server/src/main/java/com/lunchtime/models/Person.java

```diff
@@ -18,20 +18,16 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
-    @NotBlank
     @Column(name = "name")
```

Comment:
You should return `@NotBlank` annotations. This changes brake application
![image](https://user-images.githubusercontent.com/6152066/77169951-03044580-6ac3-11ea-992d-527246dfcf6d.png)

Reply:
Application did not break, and your table will be merged with task login and everything good.

Reply:
Your code forces next developers do additional work. Validation in POST method worked well, and your PR breaks it

Reply:
how it's broken, you can not add user?

Reply:
New Person must have phone number, without this number he or she can't make order. The same with password and username. Those field are required by design in every normal web application

You added additional feature in development environment (testing data) and destroyed production requirenment

Reply:
It doesn't connect with my task, it connects with whole application functionality. New functionality shouldn't destroy previous one